src/components/Country/Country.jsx

```diff
@@ -0,0 +1,109 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { setCountry } from '../../redux/country/country';
+import { getpollution, removePollution } from '../../redux/currentPollution/currentPollution';
+import Damages from '../damages/Damages';
+import Spinner from '../Loading/Loading';
+import './Country.css';
+
+function Country() {
+  const dispatch = useDispatch();
+  const continent = useSelector((state) => state.continent);
+  const country = useSelector((state) => state.country);
+  const pollution = useSelector((state) => state.pollution);
+
+  const countrySelect = (country) => {
+    setCountry(dispatch, country);
+  };
+
+  const revert = () => {
+    countrySelect({});
+    removePollution(dispatch);
+  };
+
+  const stat = {
+    1: ['Good', '#0f0'],
+    2: ['Fair', '#66ff00'],
+    3: ['Moderate', '#eeff00'],
+    4: ['Poor', '#ff7300'],
+    5: ['Very Poor', '#ff0000'],
+  };
+  useEffect(() => {
+    dispatch(getpollution);
+    dispatch(removePollution);
+  }, []);
+
+  return (
+    <div className="wholeClass">
+      <div className="navStyle">
+        <NavLink
+          to="/continents"
+          onClick={revert}
+        >
+          <span>{continent.toUpperCase()}</span>
+        </NavLink>
+        <span>{` <<<<< ${country.name}`}</span>
+      </div>
```

Comment:
- [ ] Even though you made a great effort on this, I am of the opinion that the details page design is not completely intuitive in the way the back button was implemented. Please kindly include a back button or an icon that indicates that you can go back to that page.